### ISSUE TITLE
Link control needs to set aria-disabled attribute on disabled state

### ISSUE THREAD

**Issue by davguerr** [OfficeDev/office-ui-fabric-react#2909]
Are you willing to submit a PR to fix? Yes
Requested priority: Normal 

#### Describe the issue:

When Link control is set to 'disabled', aria-disabled attribute needs to be set to 'true'.
Additionally, the link (or button) should NOT be keyboard focusable on 'disable' state

#### Actual behavior:

1. aria-disabled attribute is not being set to any value
2. the link is focusable via keyboard

#### Expected behavior:

1. aria-disabled attribute should be set to 'true'
2. the link should NOT be focusable via keyboard

**Comment by lynamemi** [comment]
The PR linked above addresses #1. #2 is a broader discussion, and we are going to keep as is for now.
